database.js

```diff
@@ -0,0 +1,31 @@
+const Database = require("better-sqlite3");
+
+const getDatabase = ({ path }) => {
+  const db = Database(path, {});
+
+  // Init table if not exists
+  db.exec(
+    "CREATE TABLE IF NOT EXISTS packages (name TEXT, version TEXT, release_date NUMBER )"
+  );
+
+  return db;
+};
+
+const getReleaseDate = ({ db, name, version }) =>
+  db
+    .prepare("SELECT release_date FROM packages WHERE name=? AND version=?")
+    .get(name, version)?.release_date;
+
+const writeReleaseDate = ({ db, name, version, releaseDate }) =>
+  db
+    .prepare(
+      "INSERT INTO packages (name, version, release_date) VALUES (?, ?, ?)"
+    )
+    .run(name, version, releaseDate.valueOf());
+
+module.exports = {
+  target: "node",
```

Comment:
Are you going to have any other targets for webpack except "node"? It seems like a NodeJS util and you can just have a single target, you don't?

Reply:
Actually no. To be honest I not so familiar with Node and had an issue to build bundle via Webpack and have tried different steps to make that work :)
Removed that out